perception/map_based_prediction/src/map_based_prediction_node.cpp

```diff
@@ -594,53 +590,56 @@ ObjectClassification::_label_type changeLabelForPrediction(
   const lanelet::LaneletMapPtr & lanelet_map_ptr_)
 {
   // for car like vehicle do not change labels
-  if (
-    label == ObjectClassification::CAR || label == ObjectClassification::BUS ||
-    label == ObjectClassification::TRUCK || label == ObjectClassification::TRAILER ||
-    label == ObjectClassification::UNKNOWN) {
-    return label;
-  } else if (  // for bicycle and motorcycle
-    label == ObjectClassification::MOTORCYCLE || label == ObjectClassification::BICYCLE) {
-    // if object is within road lanelet and satisfies yaw constraints
-    const bool within_road_lanelet = withinRoadLanelet(object, lanelet_map_ptr_, true);
-    const float high_speed_threshold = 25.0 / 18.0 * 5.0;  // High speed bicycle 25 km/h
-    // calc abs speed from x and y velocity
-    const double abs_speed = std::hypot(
-      object.kinematics.twist_with_covariance.twist.linear.x,
-      object.kinematics.twist_with_covariance.twist.linear.y);
-    const bool high_speed_object = abs_speed > high_speed_threshold;
-
-    // if the object is within lanelet, do the same estimation with vehicle
-    if (within_road_lanelet) {
-      return ObjectClassification::MOTORCYCLE;
-    } else if (high_speed_object) {
+  switch (label) {
+    case ObjectClassification::CAR:
+    case ObjectClassification::BUS:
+    case ObjectClassification::TRUCK:
+    case ObjectClassification::TRAILER:
+    case ObjectClassification::UNKNOWN:
+      return label;
+      break;
```

Comment:
unnecessary?

Reply:
I think it depends. Those objects dont have a particular implementation, but may have in the future. I think it makes no difference taking them out, but I believe it is a preference thing. If you feel it should be taken out, I will remove it.

Reply:
The reason I though this is uncesessary is because this `break` is never ecxecuted. And reader may think this is called in some condition(may be I'm thinking too much?
If you keep this, could you explain in the comment why you keep this part?

Reply:
I think you can keep it. Perhaps we might do some trick with high speed `Unknown` objects.

Reply:
@kyoichi-sugahara  after thinking about it, I think it is better to take the break out. But the rest of the code is necessary. I will remove the other breaks (that come after a return) when I have time

Reply:
@kyoichi-sugahara Done, I took out the redundant breaks